lib/index.js

```diff
@@ -69,11 +74,12 @@ AdWords.prototype.page = function() {
  */
 
 AdWords.prototype.track = function(track) {
+  var remarketing = this.options.remarketing;
   var id = this.options.conversionId;
   var events = this.events(track.event());
   var revenue = track.revenue() || 0;
   each(function(label) {
-    var props = track.properties();
+    var props = remarketing ? extend(track.properties(), window.google_custom_params) : {};
```

Comment:
Yeah I'm fairly certain that the Google API weren't doing anything with the props that we were sending if the `remarketing` flag was `false` so this wouldn't be a breaking change.